### PR TITLE
[TIMOB-19757] move use-app-thinning to ios tag

### PR DESCRIPTION
**Pull request by feons**
https://jira.appcelerator.org/browse/TIMOB-19757

### CODE REVIEW

**Comment by cb1kenobi** [comment]
Can you please update the tests too?

**Comment by feons** [comment]
@cb1kenobi, updated tests.

File: node_modules/titanium-sdk/tests/test-tiappxml.js [hunk 1]
```diff
@@ -90,6 +90,8 @@ var ti = require('../lib/titanium'),
 	console.log(tiapp.toString('pretty-json'));
 	console.log('\nXML:');
 	console.log(tiapp.toString('xml'));
+	console.log('\nIOS use-app-thinning:');
+	console.log(tiapp.ios['use-app-thinning']);
```
Comment (cb1kenobi) [inline node_modules/titanium-sdk/tests/test-tiappxml.js]: These 2 lines are not necessary.

**Comment by cb1kenobi** [comment]
Looks good! **APPROVED**

**Pull request merged**